### PR TITLE
Add awesome-k8s badge + layout updates

### PR DESCRIPTION
**Pull request by ishanmkh**
- new banner image (many iterations to optimize size and quality)
- updated layout for readability 
- added awesome-k8s

### CODE REVIEW

**Comment by ishanmkh** [comment]
- need to remove line from banner image but can do later. need compressed image form charles. his compression magic is  >>> than mine

File: README.md [hunk 1]
```diff
@@ -23,78 +29,110 @@ We're building up Pixie for broad use by the end of 2020 with an initial focus o
...
-    `bash -c "$(curl -fsSL https://withpixie.ai/install.sh)"`
 
-3. **Setup a sandbox**  You can easily build a local set-up to use Pixie if you don't already have a K8s setup or meaningful workloads to mintor set-up.
+#### Install CLI 
 
-    - Set-up a local K8s environment:
-        - [Install Kind](https://kind.sigs.k8s.io/docs/user/quick-start/) and run `kind create cluster`
-        - [Install Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) and run `minikube start`
-        - Review our [docs](https://work.withpixie.ai/docs/getting-started) for detailed instructions and alternate options.
+Copy and run the command below to install Pixie's CLI"
```
Comment (zasgar) [inline README.md:48]: extra "

File: README.md [hunk 2]
```diff
@@ -1,20 +1,26 @@
 <p align="center">
 
-   [![Pixie!](./assets/pixie_banner.svg)](https://pixielabs.ai)
-
-  <p align="center">
-    Live-debug distributed environments with <a href="https://work.withpixie.ai/">Pixie</a>
-  </p>
+  [![Pixie!](./assets/readme_banner_v2.png)](https://pixielabs.ai)
```
Comment (zasgar) [inline README.md:3]: Typo in banner "securely"

Comment (ishanmkh) [reply README.md]: updated

File: README.md (hunk 1 above)
Comment (ishanmkh) [reply README.md]: updated

**Pull request merged**